glyphs2fontir/src/source.rs

```diff
@@ -935,6 +934,23 @@ impl Work<Context, WorkId, WorkError> for GlyphIrWork {
     }
 }
 
+// this is intended as a drop-in replacement for a previous function of this
```

Comment:
The first sentence is not going to be very helpful to anyone who didn't read the prior version. Suggest delete or change it to explain what it does.